frontends/election-manager/src/lib/backends/memory_backend.ts

```diff
@@ -150,6 +150,18 @@ export class ElectionManagerStoreMemoryBackend
     );
   }
 
+  getCvrs(): Promise<CastVoteRecord[]> {
+    if (!this.electionDefinition) {
+      throw new Error('Election definition must be configured first');
+    }
+
+    if (!this.castVoteRecordFiles) {
+      return Promise.resolve([]);
+    }
+
+    return Promise.resolve([...this.castVoteRecordFiles.castVoteRecords]);
+  }
+
```

Comment:
In theory, should we be writing tests for `MemoryBackend`? I'm not suggesting you edit this PR to do so (I don't want to have to write tests for `MemoryBackend` 😄 ) but wondering if it would be good practice to. Or perhaps testing our test utilities is a rabbit hole.

Reply:
😅 yeah, I think we would ideally make sure there's test coverage for this.
I'm sort of hoping that once the dust settles on the new architecture push, we're ok with moving over to spinning up a real server (with the in-memory sqlite instance) for our functional tests and using that instead of duplicating server behaviour here. It does feel a bit annoying to maintain `MemoryBackend`, but we can evaluate the test performance hit of using a real server (which I'm guessing won't be much)

Reply:
cc @carolinemodic

As mentioned in eng planning, I'm thinking the options here are:
1. [S] Punt on reading raw CVR data from the backend and schedule some future time for moving tally computation to the backend.
2. [M] Invest in making sure this temporary API can handle large lists of CVRs (e.g. add support for paging)
3. [L?] Opportunistically move tally computation to the backend as part of this project and avoid investing in the temporary API.

Maybe (2) is a good compromise here, so we don't stay in the state of having two sources of truth for raw CVR data?

Reply:
oops, thanks! 😅
will move to the right thread